Makefile

```diff
@@ -1,13 +1,14 @@
-CXXFLAGS = -O3 -g0 -march=native
-LDFLAGS = $(CXXFLAGS)
+CXXFLAGS = -march=native -I/usr/local/opt/openssl/include
+LDFLAGS = $(CXXFLAGS) -L/usr/local/opt/openssl/lib
```

Comment:
Thesse path depends on environments?

Reply:
yes, it should be environment dependent. I'll make it configurable